packages/web-service/src/pages/applications/applications.njk

```diff
@@ -13,6 +13,8 @@
   <p class="govuk-body">Return to an existing application by selecting it from your list of licence applications.</p>
 {% endset %}
 
+{% set loggedIn = 'true' %}
```

Comment:
This should come down from the handlers as data. You can probably add a flag to additional properties which can be referenced on any page. The code can check for the signed in status from the auth cache.  `request.cache().getAuthData()`

You will probably need to clear the cache in `packages/web-service/src/pages/sign-out/sign-out.js`

Reply:
Actually you should not need to do it thinking about it. As soon as you unstate the cache is decoupled from the cookie....

Reply:
Addressed